src/main.ts

```diff
@@ -35,6 +37,18 @@ const bootstrap = async () => {
     })
   );
 
+  app.use(
+    session({
+      secret: 'keyboard cat',
+      resave: false,
+      saveUninitialized: true,
+      cookie: { secure: true },
```

Comment:
passport session needs store that can be preserved between service restart. By default the InMemory store is not a good idea. All user sessions will be lost.

Reply:
@atstojanov this is redundant and outdated, I removed it. Sessions are mandatory for different auth flows.